drip/models.py

```diff
@@ -42,6 +42,15 @@ class AbstractDrip(models.Model):
     )
     message_class = models.CharField(max_length=120, blank=True, default="default")
 
+    campaign = models.ForeignKey(
```

Comment:
Can't a drip be related to more than one Campaing?

Reply:
For now it only can be related to one campaign, we could talk about this in the future. WDYT?

Reply:
We should discuss if it does make sense a N to 1 relationship or N to N relationship. Maybe we can discuss it tomorrow. I think that since you are making this big PR, we should define this before the documentation changes